src/core/constraint.jl

```diff
@@ -180,6 +180,19 @@ function constraint_storage_complementarity(pm::GenericPowerModel, n::Int, i)
     JuMP.@constraint(pm.model, sc*sd == 0.0)
 end
 
+""
+function constraint_storage_complementarity_mixed_int(pm::GenericPowerModel, n::Int, i, charge_lb, charge_ub, discharge_lb, discharge_ub)
+    sc = var(pm, n, :sc, i)
+    sd = var(pm, n, :sd, i)
+    sc_on = var(pm, n, :sc_on, i)
+    sd_on = var(pm, n, :sd_on, i)

```

Comment:
style nit, add a space between variables and constraints.  Delicate use of spacing helps make code easier to read and understand.